src/main/java/com/puppycrawl/tools/checkstyle/checks/ArrayTypeStyleCheck.java

```diff
@@ -27,9 +27,16 @@
 /**
  * Checks the style of array type definitions.
  * Some like Java-style: {@code public static void main(String[] args)}
- * and some like C-style: public static void main(String args[])
+ * and some like C-style: {@code public static void main(String args[])}.
  *
- * <p>By default the Check enforces Java style.
+ * <p>By default the Check enforces Java style.</p>
+ *
+ * <p>This check also enforces only Java style for method return types.</p>
```

Comment:
Reword to:
`This check strictly enforces only Java style for method return types regardless of the value for 'javaStyle'.`

Please also add explanation that this is because C doesn't compile methods with array declarations on the method name.